src/Components/shareList.js

```diff
@@ -34,7 +33,7 @@ export function ShareList() {
   };
 
   return (
-    <Form onSubmit={handleSubmitShareCode} className="pa2">
+    <form onSubmit={handleSubmitShareCode} className="pa2">
```

Comment:
I like that you went with the native `form` element here. This makes this component that much less complex.